db.env

```diff
@@ -0,0 +1,4 @@
+MYSQL_DATABASE=${DB_NAME}
+MYSQL_USER=${DB_USERNAME}
```

Comment:
Since this files are not commited, good approach is to add corresponding example file. In this case add _**db.env.example**_ file and that would have the same keys, just mocked (not correct) values. That way all other team members will have knowledge what needs to be added in their .env files

Reply:
I thought I could just send the `.env` file to them. But there should still be some examples?

Reply:
The only file i didn't commit is `.env`. That's were all the values used in the other .env files will come from. Would it be enough to just make an `.env.example`?